### PR TITLE
[Workflow] Add guidelines for "how to modify c++ client api doc"

### PR DESCRIPTION
**Pull request by Anonymitaet**

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@BewareMyPower @Huanli-Meng @Jennifer88huang @sijia-w could you please help review? Thanks

File: site2/README.md [hunk 1]
```diff
@@ -211,6 +211,60 @@ If you want to change the documentation for a previous version, you can access f
 
 For more details about versioning, refer to [Versioning](https://docusaurus.io/docs/en/versioning).
 
+## C++ client API doc
+
+You can use the C++ client API to connect to Pulsar and handles data. [Pulsar C++ client API doc](https://pulsar.apache.org/api/cpp/https://pulsar.apache.org/api/cpp/) provides extensive implementation details and usage conventions for the C++ client.
```
Comment (Huanli-Meng) [inline site2/README.md:216]: connect to a Pulsar cluster?

**Review by Huanli-Meng — APPROVED** [review]
only one minor comment. PTAL

File: site2/README.md [hunk 2]
```diff
@@ -211,6 +211,60 @@ If you want to change the documentation for a previous version, you can access f
...
+   
+   > #### Tip
+   >
+   > - Each class has its corresponding `h` file. For example, if you want to update the [ProducerConfiguration class](https://pulsar.apache.org/api/cpp/classpulsar_1_1_producer_configuration.html), you need to modify the file [ProducerConfiguration.h](https://github.com/apache/pulsar/blob/master/pulsar-client-cpp/include/pulsar/ProducerConfiguration.h).
+   > 
+   > - If you want to update descriptions for member functions, you can take [Pulsar Java API doc](https://github.com/apache/pulsar/tree/master/pulsar-client-api/src/main/java/org/apache/pulsar/client/api) as references.
+
+2. Format the code file.
+   
+   Go to the directory `pulsar/pulsar-client-cpp` and run the command `clang-format -i pulsar-client-cpp/include/pulsar/<your-file-name>`.
```
Comment (BewareMyPower) [inline site2/README.md:254]: The command should be `clang-format -i include/pulsar/<your-file-name>`

**Review by BewareMyPower — CHANGES_REQUESTED** [review]

**Comment by BewareMyPower** [comment]
Since `brew install` only works on MacOS, is it better to note that these install commands (`brew install xxx`) is an example from MacOS?

File: site2/README.md (hunk 2 above)
Comment (Anonymitaet) [reply site2/README.md:254]: OK

**Comment by Anonymitaet** [comment]
> Since `brew install` only works on MacOS, is it better to note that these install commands (`brew install xxx`) is an example from MacOS?

Yes! Thanks for reminding me of this. Added.

File: site2/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (BewareMyPower) [reply site2/README.md:216]: I prefer to use `a Pulsar cluster`. But I think both are good.

[the remark this answers is not in the document]
Comment (Anonymitaet) [reply site2/README.md:216]: OK, I've updated it to `a Pulsar cluster`, thanks @Huanli-Meng @BewareMyPower

**Review by BewareMyPower — APPROVED** [review]

**Review by sijie — APPROVED** [review]

**Pull request merged**